### ISSUE TITLE
Buttons/links still not interactive outside of text

### ISSUE THREAD

**Issue by mirekys** [VSCHT/restaurovani#41]
TLDR: All buttons/links should provide clear indication that they are clickable, and should be clickable anywhere, not just by hitting exactly the label text.

- [ ] missing any interactivity indication (hover state + pointer cursor)
![Screenshot 2023-11-01 at 14 34 51](https://github.com/VSCHT/restaurovani/assets/646743/d9d69d59-fb17-48d4-a12a-c55cc2a20055)

- [ ] click outside of exactly hitting label text doesn't do anything

![Screenshot 2023-11-01 at 14 36 51](https://github.com/VSCHT/restaurovani/assets/646743/106b2944-afed-4271-9186-20d339bf1719)
![Screenshot 2023-11-01 at 14 37 00](https://github.com/VSCHT/restaurovani/assets/646743/e620a7e5-989f-482f-b7c2-9abb257eb74d)
